test/RESTAssured/src/test/java/com/backbase/smoketest/AccessControlTests.java

```diff
@@ -0,0 +1,53 @@
+package com.backbase.smoketest;
+
+import com.backbase.config.Constants;
+import com.backbase.services.services.AccessControl;
+import com.backbase.services.services.Identity;
+import org.junit.Test;
+
+public class AccessControlTests {
+
+    private final AccessControl accessControl = new AccessControl();
+    private final Identity identity = new Identity();
+
+    @Test
+    public void canGetDataGroups() {
```

Comment:
this test fails when I run it with mvn command, getting 403

Reply:
(Same as below) I have removed this test. It was flakey in the smoke testing via Postman.
Perhaps I am missing a prerequisite to have it running first time in this setup.
Thanks for checking.